lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -466,6 +479,19 @@ def _container_security_context_is_set(
         return True
 
 
+class NadConfigChangedEvent(EventBase):
+    """Event triggered when an existing network attachment definition is changed."""
+
+    def __init__(self, handle: Handle):
+        super().__init__(handle)
+
+
+class KubernetesMultusCharmEvents(CharmEvents):
+    """Kubernetes Multus Charm Events."""
+
+    nad_config_changed = EventSource(NadConfigChangedEvent)
+
+
```

Comment:
I'm not sure we want to have those here. Charm authors can use whatever even they want ultimately and having those here confuses the picture I feel.

Reply:
Event definitions are removed from the library side.